hdtv/fitxml.py

```diff
@@ -391,9 +409,13 @@ def RestoreFromXml_v1_5(self, root, sid, calibrate=False, refit=False, interacti
         allow for a more general interpretation as arbitrary fit parameters.
         No big change!
         """
-        return self.RestoreFromXml_v1_4(root, sid, calibrate, refit, interactive=interactive)
+        return self.RestoreFromXml_v1_4(
+            root, sid, calibrate, refit, interactive=interactive
+        )
 
-    def RestoreFromXml_v1_4(self, root, sid, calibrate=False, refit=False, interactive=True):
+    def RestoreFromXml_v1_4(
```

Comment:
Codacy found an issue: [FitXml.RestoreFromXml_v1_4 is too complex (16) (MC0001)](https://app.codacy.com/manual/janmayer/hdtv/pullRequest?prid=6293112)